isobus/src/isobus_virtual_terminal_client.cpp

```diff
@@ -3394,7 +3565,7 @@ namespace isobus
 							if (ready)
 							{
 								bool found = false;
-								for (AuxiliaryInputDevice &aux : parentVT->auxiliaryInputDevices)
+								for (AssignedAuxiliaryInputDevice &aux : parentVT->assignedAuxiliaryInputDevices)
 								{
 									if (aux.modelIdentificationCode == modelIdentificationCode)
```

Comment:
If this is true, should we `break;` below?

Reply:
Switched to `std::find_if`